structure/structure.py

```diff
@@ -1,25 +1,54 @@
+import os
 import sys
 
+from griptape.artifacts import TextArtifact
 from griptape.drivers import (
     GriptapeCloudEventListenerDriver,
 )
 from griptape.events import (
     EventListener,
+    FinishStructureRunEvent,
 )
 from griptape.structures import Agent
 from griptape.tools import Calculator
 
+
+# Returns True if this program is being run from within Griptape Cloud or via the Skatepark emulator
+def is_running_in_managed_environment():
+    return "GT_CLOUD_RUN_ID" in os.environ
+
+
 input = sys.argv[1]
 
+# We need an event driver to communicate events from our program back to Skatepark
+event_driver = GriptapeCloudEventListenerDriver(
+    base_url="http://127.0.0.1:5000", api_key="..."
```

Comment:
should this be pulled from an env somewhere, or is it always going to be 127.0.0.1:5000?

Reply:
We need to guarantee that it's there...we could default it if it's not found in the `.env`, so it's more like an override?

Reply:
Yeah, having an override would be great. I know a customer could build that in, but since this is a template we might want to handle that for them.